modules/cluster-logging-uninstall.adoc

```diff
@@ -60,9 +60,33 @@ Do not delete the `openshift-operators-redhat` project if other global operators
 
 .. Confirm the deletion by typing `openshift-operators-redhat` in the dialog box and click *Delete*.
 
-. Optional: Remove any OpenShift Logging persistent volume claims (PVC):
+.Next steps
+
+Uninstalling OpenShift Logging does not remove the persistent volume claims (PVCs). To preserve or delete the PVCs, persistent volumes (PVs), and associated data, you must take further action.
+
+
+[WARNING]
+====
+Releasing or deleting PVCs can delete PVs and cause data loss.
+====
+
+. Evaluate whether to keep the PVCs for reuse with other pods. If so, keep the labels or PVC names you need to reclaim the PVCs.
+
+. Optional: If you do not want to reclaim the PVCs, you can delete them.
++
+If a PV's `persistentVolumeReclaimPolicy` is `delete`, deleting the PVC deletes its PV. This action releases the backing storage and removes all data, such as collected log data.
```

Comment:
You could state what type of object `persistentVolumeReclaimPolicy` is, within the sentence. For example:

"If a PV's `persistentVolumeReclaimPolicy` parameter is set to `delete`".